### DISCUSSION TITLE
Presets

### DISCUSSION THREAD

**Discussion by louiss0** [nxtensions/nxtensions#346] [Q&A] [answered]
I don't know how to use the presets created by this plugin. How do I use them?  The thing is that an app is being created by this plugin. How am I supposed to refer to it in the workspace being created I don't understand what is going on.

**Comment by leosvelperez** [comment, marked as answer]
Hey @louiss0!

An Nx Preset is a special generator that you can use when creating a workspace. Using the `@nxtensions/astro` preset when creating a workspace will generate a workspace with an Astro application. This is mentioned in [the README](https://github.com/nxtensions/nxtensions/blob/main/packages/astro/README.md#create-a-new-nx-workspace-with-an-astro-application). You can also read [the Nx docs discussing presets](https://nx.dev/recipes/advanced-plugins/create-preset#preset).

So, the `@nxtensions/astro` plugin doesn't create presets. It contains a preset to create an Nx workspace with an Astro application. I think there might be a misunderstanding of what a preset is. Please refer to the docs mentioned above.
